### PR TITLE
GameDB : critical changes for the upcoming 1.6 stable version.

### PR DESCRIPTION
**Pull request by atomic83GitHub**
A short version of the closed pull request with some new adds, this was made in order to save the time of the devs and for avoid users to manualy add these folowing hacks.

I had test them but I would like to have some other tests on these games before do anything else or merge the PR.

### CODE REVIEW

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1263,7 +1269,8 @@ Serial = SCES-50354
 Name   = Klonoa 2 - Lunatea's Veil
 Region = PAL-M5
 Compat = 5
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3	//Objects needed appear in wrong places without it
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Trailing whitespace. It's not consistent with the others.

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -34503,7 +34582,8 @@ Serial = SLUS-20151
 Name   = Klonoa 2 - Lunatea's Veil
 Region = NTSC-U
 Compat = 5
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3	//Objects needed appear in wrong places without it
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -42764,7 +42863,8 @@ Region = NTSC-U
 Serial = SLUS-28004
 Name   = Klonoa 2 - Lunatea's Veil [Trade Demo]
 Region = NTSC-U
-eeClampMode = 3		//Objects needed appear in wrong places without it
+eeClampMode = 3	//Objects needed appear in wrong places without it
```
Comment (lightningterror) [inline bin/GameIndex.dbf]: Ditto.

File: bin/GameIndex.dbf (hunk 1 above)
Comment (atomic83GitHub) [reply bin/GameIndex.dbf]: It should be fixed now.

Comment (lightningterror) [reply bin/GameIndex.dbf]: It's still not fixed. Copy my line and replace it, that should fix it up.

**Comment by lightningterror** [comment]
Something turtleli taught me, you can add [skip ci] to the commit description to skip building on appveyor/travis since Gamedb isn't built. :smile:

**Comment by atomic83GitHub** [comment]
It is fixed now, but I can't modify comments on my previous commits, how should I do?

**Comment by MrCK1** [comment]
Are you talking about the commit messages" Then you need to do a rebase. Are you using a GUI or shell to push your commits to Hit?

**Comment by atomic83GitHub** [comment]
The Github desktop.

**Comment by MrCK1** [comment]
1. OK, go to the repository tab and click "Open in Powershell" (If you get prompted to install Git, you'll have to do that first)

2. Once that's open, type in `git rebase -i HEAD~3`

3. Notepad will pop up now, where it says `pick` - change the bottom 2 to `squash` so that it looks like this:

image


4. After you make those changes, save the file and close the window. You're going to see another window that pops up. 

5. What you'll do then is delete the last two commit messages and keep the first one. Then you can reword it however you want

image

6. After you edit the commit messages, save the file and close it.

7. Go back to Powershell and type in `git push --force`

**Comment by atomic83GitHub** [comment]
Oops, I do a wrong thing wich has destroyed the entire DB, I am going to close the PR and create a another one.

**Comment by MrCK1** [comment]
Just a tip for the future, you don't have to delete an entire branch and restart if you make a mistake. (I remember when I would do the same thing :P)

Git has other commands that will help you revert those mistakes without having to redo your work ;).

**Pull request closed without merging**